arm/Microsoft.ServiceBus/namespaces/deploy.bicep

```diff
@@ -200,6 +210,16 @@ module serviceBusNamespace_virtualNetworkRules 'virtualNetworkRules/deploy.bicep
   }
 }]
 
+module serviceBusNamespace_NetworkAcls 'networkRuleSets/deploy.bicep' = if (skuName == 'Premium') {
```

Comment:
Also, not sure about the condition on this. I am thinking I would like deployments to fail if it's on another sku and providing this option. Assuming I configured my param here to be skuName = 'Standard' and provided network rules, I would have a param file that would be "lying", as the network rules would never be deployed. Feel this would be breaking with a "single source of truth" principle that IaC solutions should have. Or am I missing something?

Reply:
@MariusStorhaug - Happy for this to be reviewed, the reason for choosing SKU is that this is only available on the premium SKU if it is passed in on the standard SKU the deployment will fail regardless of 'True or False' selection.
A possible change could be to `if (allowTrustedServices == False) as the default behaviour is to set to 'True'. Thoughts?